src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java

```diff
@@ -22,14 +22,19 @@
   public void prepareTest() {
     driver.manage().timeouts().pageLoadTimeout(100, TimeUnit.SECONDS);
   }
-  
-  private static final String[]
-      DIFFERENT_HUBS_WIKIS =
-      {"destiny", "cocktails", "thehungergames", "dc", "tardis", "starwars", "lego"};
-  private static final String[]
-      DIFFERENT_HUBS_COLORS =
-      {"#94d11f", "#8ca038", "#ff7f26", "#ff5400", "#00b7e0", "#09d3bf",
-       "#ffd000"};
+
+  /**
+   * Wiki name, Main page, HUB color
+   */
+  private static final String[][] WIKIS = {
+      {"destiny", "Destiny_Wiki", "#94d11f"},
```

Comment:
What do you think if instead of using hex color values as string make an `enum` which will have color names and hex color values?
It will be more readable and reusable :wink: